### PR TITLE
ISD search results ordering

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/TT-1408)
[this task](https://uktrade.atlassian.net/browse/TT-1438)

### CODE REVIEW

File: company/search.py [hunk 1]
```diff
@@ -69,16 +71,16 @@ class CompanyDocument(Document):
...
+            'title': field.Text(copy_to='casestudy_wildcard'),
+            'short_summary': field.Text(copy_to='casestudy_wildcard'),
+            'description': field.Text(copy_to='casestudy_wildcard'),
+            'sector': field.Text(copy_to='casestudy_wildcard'),
+            'keywords': field.Text(copy_to='casestudy_wildcard'),
+            'image_one_caption': field.Text(copy_to='casestudy_wildcard'),
+            'image_two_caption': field.Text(copy_to='casestudy_wildcard'),
+            'image_three_caption': field.Text(copy_to='casestudy_wildcard'),
+            'testimonial': field.Text(copy_to='casestudy_wildcard'),
+            'slug': field.Text(copy_to='casestudy_wildcard'),
```
Comment (kowalcj0) [inline company/search.py]: @richtier it doesn't look like this will fix https://uktrade.atlassian.net/browse/TT-1438

Comment (richtier) [reply company/search.py]: it does. I even wrote a test to confirm :boom: https://github.com/uktrade/directory-api/pull/514/files#diff-ab4b804cfb7c052c57aa42630c7940cfR1882

File: company/search.py [hunk 2]
```diff
@@ -13,14 +29,14 @@ class CaseStudyFieldsMixin:
     title = field.Text(copy_to='wildcard')
     short_summary = field.Text(copy_to='wildcard')
     description = field.Text(copy_to='wildcard')
-    sector = field.Text(copy_to='wildcard')
+    sector = field.Text(copy_to='wildcard',)
```
Comment (kowalcj0) [inline company/search.py]: is `,` necessary here?
None of the other `Text` field definitions has it.

File: company/search.py [hunk 3]
```diff
@@ -13,14 +29,14 @@ class CaseStudyFieldsMixin:
...
     keywords = field.Text(copy_to='wildcard')
     image = field.Text(index=False)
     company_number = field.Text(index=False)
     image_one_caption = field.Text(copy_to='wildcard')
     image_two_caption = field.Text(copy_to='wildcard')
     image_three_caption = field.Text(copy_to='wildcard')
-    testimonial = field.Text(copy_to='wildcard')
+    testimonial = field.Text(copy_to='wildcard',)
```
Comment (kowalcj0) [inline company/search.py]: ditto

File: company/search.py [hunk 4]
```diff
@@ -35,10 +51,12 @@ class CaseStudyInnerDoc(CaseStudyFieldsMixin, InnerDoc):
...
 
 class CompanyDocument(Document):
-    wildcard = field.Text()
+    wildcard = field.Text(analyzer=american_english_analyzer)
+    casestudy_wildcard = field.Text(analyzer=american_english_analyzer)
+
     case_study_count = field.Integer()
     date_of_creation = field.Date(index=False)
-    description = field.Text(copy_to='wildcard')
+    description = field.Text(copy_to='wildcard',)
```
Comment (kowalcj0) [inline company/search.py]: ditto

File: company/search.py [hunk 5]
```diff
@@ -49,7 +67,7 @@ class CompanyDocument(Document):
     has_single_sector = field.Boolean()
     modified = field.Date(index=False)
     name = field.Text(copy_to='wildcard')
-    number = field.Text(copy_to='wildcard')
+    number = field.Text(copy_to='wildcard',)
```
Comment (kowalcj0) [inline company/search.py]: same here

File: company/search.py (hunk 1 above)
Comment (kowalcj0) [reply company/search.py]: :1st_place_medal:

**Pull request merged**